bots/pr/src/main/java/org/openjdk/skara/bots/pr/LimitedCensusInstance.java

```diff
@@ -39,7 +44,7 @@ static Optional<LimitedCensusInstance> createLimitedCensusInstance(HostedReposit
             Optional<JCheckConfiguration> configuration = jCheckConfiguration(hostedRepositoryPool,
                     repository, ref, confOverrideRepo, confOverrideName, confOverrideRef);
             if (configuration.isEmpty()) {
-                return Optional.empty();
+                throw new InvalidJCheckConfException(".jcheck/conf is invalid or missing");
```

Comment:
Hi Erik, I don't know if I understood you exactly to throw the exception here.

Reply:
I think it would fit better in `configuration` as an `.orElseThrow()`. In that case it's only thrown if jcheck is missing, so the exception should be named MissingJCheckConfException.

The actual parsing of the configuration seems to only explicitly throw IllegalArgumentException. It's possible that other parse problems would result in other runtime exceptions, such as NPE. I think we should wrap the parse call with try-catch for RuntimeException and throw an `InvalidJCheckConfException` if that happens.